moped-editor/src/views/projects/projectsListView/ProjectsListViewFiltersConf.js

```diff
@@ -337,6 +337,21 @@ export const PROJECT_LIST_VIEW_FILTERS_CONFIG = {
       "string_is_not_null",
     ],
   },
+  substantial_completion_date: {
+    name: "substantial_completion_date",
+    label: "Substantial completion date",
+    placeholder: "Substantial completion date",
+    type: "date",
+    defaultOperator: "date_equals",
+    operators: [
+      "date_equals",
+      "date_greater_than",
+      "date_greater_than_equal_to",
+      "date_less_than",
+      "date_less_than_equal_to",
+      "date_is_null",
```

Comment:
beyond in the scope of this PR, but reading through this code made me realize we don't have an "is not blank" operator for dates. I created [an issue](https://github.com/cityofaustin/atd-data-tech/issues/19470) but it isn't a priority.